x-pack/test/functional/services/ml/data_frame_analytics_edit.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { MlCommon } from './common';
+import { MlApi } from './api';
+
+export function MachineLearningDataFrameAnalyticsEditProvider(
+  { getService }: FtrProviderContext,
+  mlCommon: MlCommon,
+  mlApi: MlApi
```

Comment:
I think the API service is currently not needed here.

Reply:
Removed in a2b564a0e7a8c95c3b34076577ed22a8b02470eb